packages/kerosene/src/string/replaceAll.ts

```diff
@@ -0,0 +1,11 @@
+const replaceAll = (replaceThis: string, withThis: string, inThis: string): string => {
```

Comment:
Is `replaceAll` necessary when we have `haystack.replace(/needle/g, replacement)`?

Reply:
technically none of these atomic utils are necessary, but creating a function allows the developer to never need to think of the implementation details.

Plus, putting this in a function makes for nicer FP `.map`-ing